pubsub/rabbitpubsub/rabbit.go

```diff
@@ -142,7 +162,15 @@ type TopicOptions struct{}
 
 // SubscriptionOptions sets options for constructing a *pubsub.Subscription
 // backed by RabbitMQ.
-type SubscriptionOptions struct{}
+type SubscriptionOptions struct {
+	// Qos properties
+	Qos *Qos
+}
+
+// Qos options to be used when we create a subscription.
```

Comment:
Let's drop this extra struct, just put PrefetchCount in SubscriptionOptions.

Reply:
Simplified